docs/en/03_Upgrading/01_Upgrading_project.md

```diff
@@ -25,9 +25,9 @@ How easy will it be to update my project? It's a fair question, and sometimes a
 
 There are some fundamental changes in SilverStripe 4:
 
-* PHP 5.6 is now the minimum required version and up to PHP 7.3 is supported. We strongly recommend you only use [supported versions of PHP](https://www.php.net/supported-versions.php).
+* PHP 7.1 is now the minimum required version for SilverStripe 4.5.0 and above and up to PHP 7.3 is supported. We strongly recommend you only use [supported versions of PHP](https://www.php.net/supported-versions.php). Note: SilverStripe 4.0.0 through 4.4.0 still support PHP 5.6.
 * SilverStripe is now even more modular which allows you to remove functionality your project might not need.
-* Common functionality sets can now be installed via SilverStripe _recipes_.
+* Common functionality sets can now be installed via SilverStripe _recipes_
```

Comment:
Sorry about this; for some reason Githubs editor does’t like Safari on iPad so I wasn’t able to fix this until submitting the PR (will have to get to PC to fix).